buildstockbatch/localdocker.py

```diff
@@ -114,19 +173,22 @@ def run_building(cls, project_dir, buildstock_dir, weather_dir, docker_image, re
             if not os.path.exists(dir_to_make):
                 os.makedirs(dir_to_make)
 
+        if cfg.get('baseline', dict()).get('custom_gems', False):
+            mnt_custom_gem_dir = '/var/oscli/gems'
+            docker_volume_mounts[os.path.join(buildstock_dir, '.custom_gems')] = {'bind': mnt_custom_gem_dir, 'mode': 'ro'}  # noqa E501
+
         osw = cls.create_osw(cfg, n_datapoints, sim_id, building_id=i, upgrade_idx=upgrade_idx)
 
         with open(os.path.join(sim_dir, 'in.osw'), 'w') as f:
             json.dump(osw, f, indent=4)
 
         docker_client = docker.client.from_env()
-        args = [
-            'openstudio',
-            'run',
-            '-w', 'in.osw',
-        ]
+        run_cmd = 'openstudio run -w in.osw'
+        if cfg.get('baseline', dict()).get('custom_gems', False):
+            run_cmd = f'openstudio --bundle {mnt_custom_gem_dir}/Gemfile --bundle_path {mnt_custom_gem_dir} ' \
+                       '--bundle_without native_ext run -w in.osw --debug'
```

Comment:
You'll need to make other changes to make this work, but yeah.

```suggestion
            run_cmd = [
                'openstudio',
                '--bundle', f'{mnt_custom_gem_dir}/Gemfile',
                '--bundle_path', mnt_custom_gem_dir,
                '--bundle_without native_ext',
                'run', '-w', 'in.osw', '--debug'
            ]
```